src/VisualStudio/Setup/Roslyn.VisualStudio.Setup.csproj

```diff
@@ -305,4 +311,4 @@
     <PackageReference Include="Microsoft.CodeAnalysis.FlowAnalysis.Utilities" Version="$(MicrosoftCodeAnalysisFlowAnalysisUtilitiesVersion)" PrivateAssets="all" />
   </ItemGroup>
   <Import Project="$(RepositoryEngineeringDir)targets\GenerateServiceHubConfigurationFiles.targets" />
-</Project>
```

Comment:
I assume this is a style-cop warning, since I didn't change that line, I used github to edit the file, and the master branch also doesn't show a newline there.

Reply:
Various editors like to add or remove the newline depending on their opinion of what is correct. You can ignore it.